source/kernel/src/servers.rs

```diff
@@ -0,0 +1,100 @@
+//! Servers
+//!
+//! This module contains servers that may be useful for systems
+
+use core::time::Duration;
+
+use crate::{
+    drivers::serial_mux::{PortHandle, WellKnown},
+    tracing, Kernel,
+};
+
+//
+// Sermux Loopback
+//
+
+/// Loopback Settings
+#[derive(Debug, Clone)]
+pub struct SermuxLoopbackSettings {
```

Comment:
tiny nit, not actually important but i thought i'd mention it given that you've put `#[non_exhaustive]` on the `WellKnown` enum: maybe this should have a private `()` field, so that it can only be constructed as
```rust
SermuxLoopbackSettings {
    port: 0,
    buffer_size: 128,
    ...Default::default(),
}
```
so that if we add more settings in the future, existing code doesn't break, but it's explicit that it's getting the defaults for any new settings?